src/platform/utilities/feature-toggles/feature-toggle-query-list.json

```diff
@@ -0,0 +1,3 @@
+{
+    "featureToggleQueryList": ["facilityLocatorShowCommunityCares"]
```

Comment:
After we add the feature toggle to the local environment Flipper admin, we'll start here and add the feature toggle string to this list. The Flipper API currently doesn't have a "get all" query so we have to tell it which feature toggle values we want returned.